.sandbox-ignore

```diff
@@ -0,0 +1,8 @@
+.git
+.github
+.gitignore
+.sandbox-config
+.sandbox-ignore
+sandbox.sh
+vendor
+node_modules
```

Comment:
Could we add `*.zip` to this ignore?

```suggestion
node_modules
*.zip
```

Also on my time sync'ing, this repo's README.md is pushed up.